### PR TITLE
Kill browserhtml

### PR DESCRIPTION
**Pull request by paulrouget**
Fixes https://github.com/servo/servo/issues/19971

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/19975)

### CODE REVIEW

**Comment by paulrouget** [comment]
Taskclusters says: "It looks like rustup is not installed."

**Comment by mbrubeck** [comment]
Looks good to me, can land with r=mbrubeck if no further changes are needed.

**Comment by cbrewster** [comment]
What is the plan for nightly builds? Will there be no browser UI for now?

**Comment by paulrouget** [comment]
> What is the plan for nightly builds? Will there be no browser UI for now?

See https://groups.google.com/forum/#!topic/mozilla.dev.servo/U5zhsBLXovM

No "real" UI, but full keyboard control. I'm porting ServoShell's minimal UI: https://github.com/paulrouget/servoshell#mini-ui

**Comment by paulrouget** [comment]
@mbrubeck thanks - I added a few more comments in an early commit, and added another extra commit to bring crash reports up to the embedder.

**Comment by paulrouget** [comment]
Fixed a typo.

**Comment by paulrouget** [comment]
@mbrubeck r?

File: components/compositing/compositor_thread.rs [hunk 1]
```diff
@@ -146,6 +146,8 @@ pub enum EmbedderMsg {
     LoadStart(TopLevelBrowsingContextId),
     /// The load of a page has completed
     LoadComplete(TopLevelBrowsingContextId),
+    /// A pipeline panicked.
+    Panic(TopLevelBrowsingContextId, String, Option<String>),
```
Comment (emilio) [inline components/compositing/compositor_thread.rs]: Please mention what the string args are, or use a named enum like:

```rust
Panic { id: TopLevelBCId, reason: String, backtrace: Option<String> }
```

**Review by emilio — APPROVED** [review]
r=me on that last commit with that nit fixed.

**Comment by jdm** [comment]
```
  ▶ OK [expected TIMEOUT] /fullscreen/api/element-request-fullscreen-not-allowed.html

  ▶ Unexpected subtest result in /fullscreen/api/element-request-fullscreen-not-allowed.html:
  │ FAIL [expected TIMEOUT] Element#requestFullscreen() when not allowed to request fullscreen
  │   → assert_equals: event.target expected Element node <div id="log"></div> but got Document node with 2 children
  │ 
  │ @http://web-platform.test:8000/fullscreen/api/element-request-fullscreen-not-allowed.html:11:5
  │ Test.prototype.step@http://web-platform.test:8000/resources/testharness.js:1494:20
  └ Test.prototype.step_func_done/<@http://web-platform.test:8000/resources/testharness.js:1534:17

  ▶ Unexpected subtest result in /html/semantics/embedded-content/the-iframe-element/iframe-allowfullscreen.html:
  │ FAIL [expected PASS] iframe-allowfullscreen
  │   → assert_true: Top level document has fullscreen enabled flag set expected true got false
  │ 
  │ @http://web-platform.test:8000/html/semantics/embedded-content/the-iframe-element/iframe-allowfullscreen.html:22:5
  │ Test.prototype.step@http://web-platform.test:8000/resources/testharness.js:1494:20
  │ async_test@http://web-platform.test:8000/resources/testharness.js:528:13
  └ @http://web-platform.test:8000/html/semantics/embedded-content/the-iframe-element/iframe-allowfullscreen.html:13:3

  ▶ Unexpected subtest result in /html/semantics/embedded-content/the-iframe-element/iframe-allowfullscreen.html:
  │ FAIL [expected PASS] iframe-noload-noallowfullscreen
  │   → assert_false: Fullscreen should not be enabled without allowfullscreen attribute expected false got true
  │ 
  │ @http://web-platform.test:8000/html/semantics/embedded-content/the-iframe-element/iframe-allowfullscreen.html:44:7
  │ test_allowfullscreen_noload@http://web-platform.test:8000/html/semantics/embedded-content/the-iframe-element/iframe-allowfullscreen.html:38:5
  │ @http://web-platform.test:8000/html/semantics/embedded-content/the-iframe-element/iframe-allowfullscreen.html:43:5
  │ Test.prototype.step@http://web-platform.test:8000/resources/testharness.js:1494:20
  │ test@http://web-platform.test:8000/resources/testharness.js:511:9
  └ @http://web-platform.test:8000/html/semantics/embedded-content/the-iframe-element/iframe-allowfullscreen.html:42:3
```

**Comment by paulrouget** [comment]
The problem is in the `window.is_top_level` function. Returning `parent_info.is_some()` instead of `parent_info.is_none()`.

**Pull request merged**